book.rb

```diff
@@ -0,0 +1,15 @@
+class Book
+  has_many :rentals
+
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals << rental
+  end
```

Comment:
- [ ] As mentioned in my previous comment the `add_rental` method in the book class here should follow a similar pattern to the one in the person class and use the `self` reference when creating a rental as shown below:
```rb
  def add_rental(person, date)
    Rental.new(date, self, person)
  end
```